test/plugins/collections.test.js

```diff
@@ -325,6 +376,15 @@ describe('collection plugin test', () => {
             })
         );
 
+        it('returns 200 when no pipelineIds in payload', () => {
+            delete options.payload.pipelineIds;
+
+            return server.inject(options).then((reply) => {
```

Comment:
Don't you also want to assert that the collection still has the pre-existing list of pipelineIDs?